services/frontend-v3/src/components/dashboardGraphes/DashboardGraphes.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import DashboardGraphesView from "./DashboardGraphesView";
+import { injectIntl } from "react-intl";
+
+class DashboardGraphes extends Component {
+  topFiveSkills(data) {
+    const locale = this.props.intl.formatMessage({ id: "language.code" });
+
+    // Skills Count:
+    let topFiveSkills = data;
+    topFiveSkills = topFiveSkills.map(skill => {
+      return {
+        name: skill.description[locale],
+        count: skill.count
+      };
+    });
+
+    return topFiveSkills;
+  }
+
+  topFiveCompetencies(data) {
+    const locale = this.props.intl.formatMessage({ id: "language.code" });
+
+    // Competencies Count:
+    let topFiveCompetencies = data;
+    topFiveCompetencies = topFiveCompetencies.map(comp => {
+      return {
+        name: comp.description[locale],
+        count: comp.count
+      };
+    });
+
+    return topFiveCompetencies;
+  }
+
+  topFiveDevelopmentGoals(data) {
+    const locale = this.props.intl.formatMessage({ id: "language.code" });
+
+    // Development Goals Count:
+    let topFiveDevelopmentGoals = data;
+    topFiveDevelopmentGoals = topFiveDevelopmentGoals.map(goal => {
+      return {
+        name: goal.description[locale],
+        count: goal.count
+      };
+    });
+
+    return topFiveDevelopmentGoals;
+  }
```

Comment:
Are all these not the same function with a different data source?

Reply:
Yeah, exactly.

Reply:
I think I got an idea to fix it, I'll try it out and post.

Reply:
Fixed it!